bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -463,6 +463,24 @@ def set_base_velocity(
 
         return cmd_id
 
+    def get_global_from_local_based_on_input_T(
+        self, x_pos, y_pos, yaw, curr_x, curr_y, curr_yaw
+    ):
+        """Transform the point given the x,y,yaw location"""
+        coors = np.array([x_pos, y_pos, 1.0])
+        local_T_global = self._get_local_T_global(curr_x, curr_y, curr_yaw)
+        x, y, w = local_T_global.dot(coors)
```

Comment:
Should it be 

```suggestion
        x, y, z = local_T_global.dot(coors)
```
??